packages/nuxt/src/generators/init/init.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { readJson, Tree } from '@nx/devkit';
+import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
+import { nuxtInitGenerator } from './init';
+
+describe('init', () => {
+  let tree: Tree;
+
+  beforeEach(() => {
+    tree = createTreeWithEmptyWorkspace();
+  });
+
+  it('should add vue dependencies', async () => {
```

Comment:
This one is a nit, but given other changes likely need to be made...

```suggestion
  it('should add vue and nuxt dependencies', async () => {
```